### ISSUE TITLE
Paths

### ISSUE THREAD

**Issue by maximromanov** [althurayya/althurayya.github.io#6]
Place that route line on top of everything --- now it is a bit difficult to see the route as it is covered by other places.

**Comment by maximromanov** [comment]
Can we also gray everything out, so that the red path is more clearly visible?

**Comment by Masoumeh** [comment]
OK. Will do. 
We also have to decide how the previous state changes when the input values changes or the user changes the active tab. This is an issue that needs to be discussed.

**Comment by maximromanov** [comment]
True: I think the view should change depending on a mode (active tab)

**Comment by Masoumeh** [comment]
But in some use cases the tabs are related, like search and resources or info tab and resources  and so on.

**Comment by maximromanov** [comment]
True: so there should be less modes than tabs:
- Selected Red, the Rest is Gray: Regions, `Route Sections`, `Find Path`

(I wonder if we should put Location and Sources into one)

**Comment by Masoumeh** [comment]
OK. But I think more cases will come up. 
Now, let's do it the way you say and when an issue happens we can change it if needed. 
About the Location tab:
I don't like it this way as well, but the point is that we have also English resources on that tab. Even if we cut down on the technical info (which I think we should), we have the Eng. sources still.

**Comment by maximromanov** [comment]
Probably we could have location tab in the following format:

Location
-- English (full text)
-- Sources (folded)
-- Technical Info (on the very bottom)

or maybe we could move the technical info somewhere else altogether...
